### ISSUE TITLE
Allow extendType & extendInputType to extend a GraphQL Type

### ISSUE THREAD

**Issue by jasonkuhrt** [graphql-nexus/nexus#673]
```ts
import { GraphQLObjectType } from 'graphql'
import { extendType } from '@nexus/schema' 

const A = new GraphQLObjectType()

const B = extendType({
	type: 'A',
	// ...
})

makeSchema({
	types: [A,B]
})
```

**Comment by jasonkuhrt** [comment]
@tgriesser WDYT?

**Comment by tgriesser** [comment]
Does this not already work?

**Comment by jasonkuhrt** [comment]
@Weakky has more context than me, apparently not.

**Comment by tgriesser** [comment]
I'm surprised it doesn't, this is the whole reason we walk & unwrap the types as `.toConfig` - specifically so `extendType` will work - guess it'd be good to start from a failing test case.

**Comment by tgriesser** [comment]
Ah, yep this is a bug.

**Comment by tgriesser** [comment]
This will be fixed by #983
